1-org/terraform.example.tfvars

```diff
@@ -30,6 +30,14 @@ default_region = "australia-southeast1"
 
 scc_notification_name = ""
 
+dns_default_region1 = "us-central1"
```

Comment:
Can we remove these from org TF vars now?

Reply:
removed